extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -505,7 +505,7 @@ angular.module('PaperUI.controllers.configuration',
 		} else {
 		    thing.item = {};
         }
-		thingSetupService.update(thing, function() {
+		thingService.updateConfig({thingUID:thing.UID},thing.configuration, function() {
```

Comment:
Yes I tried them, even with different profiles but they doesn't seem to work. So normally I just use the keyboard shortcut.